provision/infinispan/create_ispn_clusters.sh

```diff
@@ -0,0 +1,357 @@
+#!/bin/bash
+
+set -e
+set -x
```

Comment:
We should surround this with the same check as other riles:

```bash
if [[ "$RUNNER_DEBUG" == "1" ]]; then
  set -x
fi
```

Reply:
Thank you - updated the PR.